jest.config.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
   transform: {
     '^.+\\.[tj]s$': 'ts-jest'
   },
-  moduleFileExtensions: ['ts', 'js', 'json']
+  moduleFileExtensions: ['ts', 'js', 'json'],
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.ts']
```

Comment:
Is there some description of what we are going to do within the task? I think so, and there are no jest lifecycle improvements in the ticket

Reply:
But are these the improvements you ask for when you've suggested removing `reflect-metadata`, aren't they?

Reply:
Well, I asked you remove the concrete line in the concrete file where it is useless https://github.com/NeuraLegion/nexploit-cli/pull/349#discussion_r968785793